app/Models/User/NewUserOrganizationRealtionship.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Models\User;
+
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\Relation;
+use App\Models\Organization\Organization;
+use App\Models\User\RoleUser;
+
+class NewUserOrganizationRealtionship extends Relation
```

Comment:
rename to UserOrganizationRelationship,

Reply:
@bradflood It has been renamed as you seen in the following URL:

https://github.com/faithcomesbyhearing/dbp/pull/924/files#diff-96492e7529d36eafa3ba7a40b9d706a5b08a41a1122de2922556ed1c0ccd9865R3